multiversx_sdk/abi/serializer_test.py

```diff
@@ -339,6 +340,14 @@ def test_deserialize():
 
         serializer.deserialize("0100", [destination])
 
+    # counted-variadic<u8>, should fail
+    destination = CountedVariadicValues(
```

Comment:
Maybe add a test with `VariadicValues` that follow the `CountedVariadicValues`?

Reply:
added unit test